src/components/badge/badge.scss

```diff
@@ -0,0 +1,17 @@
+:host {
+  display: block;
+}
+
+.badge-container {
+    display: inline-block;
+    font-size: 0.7rem;
+    min-width: 1.2rem;
+    height: 1.2rem;
+    border-radius: 1.2rem;
+    line-height: 1.2rem;
+    text-align: center;
+    background-color: #EEF1F5;
```

Comment:
It would be nice if this was configurable. We should use a CSS variable for it :slightly_smiling_face: 
We might need one for the color of text as well